lxd/instance/drivers/driver_lxc.go

```diff
@@ -2330,12 +2330,18 @@ func (c *lxc) detachInterfaceRename(netns string, ifName string, hostName string
 func (c *lxc) Start(stateful bool) error {
 	var ctxMap log.Ctx
 
-	// Setup a new operation
-	op, err := operationlock.Create(c.id, "start", false, false)
-	if err != nil {
-		return errors.Wrap(err, "Create container start operation")
+	// Get current or set a new operation
```

Comment:
I see a similar pattern repeatedly through this commit, I wonder if this couldn't be abstracted into a function in `operationlock` that we can just call everywhere where we have this get or set pattern.